club/templates/club/base.html

```diff
@@ -0,0 +1,36 @@
+<!-- {% load static %} -->
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```

Comment:
I recommend using this plugin: https://www.youtube.com/watch?v=YmJ0TACm8oE

Reply:
ah but maybe not, seems you already have: https://pypi.org/project/crispy-bootstrap5/

What I like about django-bootstrap5 though is that you don't have to hardcode these links ...